apps/omg_child_chain_rpc/config/config.exs

```diff
@@ -25,7 +25,7 @@ config :phoenix,
 config :omg_child_chain_rpc, OMG.ChildChainRPC.Tracer,
   service: :web,
   adapter: SpandexDatadog.Adapter,
-  disabled?: {:system, "METRICS", false},
+  disabled?: {:system, "DD_DISABLED", false, {String, :to_existing_atom}},
```

Comment:
maybe when we're at it, let's make sure `docs/manual_.....md` has the `mix...` invocations that conform to these, recently I and others couldn't get that running